src/reference/workers/generic-worker/features.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Features
+order: 10
+---
+
+Features are capabilities that can be enabled in the generic worker for use by
+a task.
+
+These features are enabled by declaring them within the task payload in the
+`features` object.
+
+Note: Some features require additional information within the task definition.
+Features may also require scopes.  Consult the documentation for each feature
```

Comment:
Also, double spaces do not render in HTML, so we need to break the habit of double spacing after a full stop. :laughing: